openedx/core/djangoapps/course_roles/models.py

```diff
@@ -0,0 +1,104 @@
+"""
+Models for course roles schema
+"""
+from django.contrib.auth import get_user_model
+from django.db import models
+
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
+from organizations.models import Organization
+
+User = get_user_model()
+
+
+class CourseRolesRole(models.Model):
+    """
+    Model for a course roles role.
+
+    A role is a collection of permissions that can be assigned to a user.
+    The services field defines in which service UI the role is intended to be assigned, such as CMS and/or LMS.
+
+    """
+    name = models.CharField(max_length=255)
```

Comment:
Would it be useful to have some kind of identifier code here that is independent of the human readable name?